router/usecase/optimized_routes_test.go

```diff
@@ -674,6 +674,143 @@ func (s *RouterTestSuite) TestGetCustomQuote_GetCustomDirectQuote_Mainnet_UOSMOU
 	s.validateExpectedPoolIDOneRouteOneHopQuote(quote, expectedPoolID)
 }
 
+// TODO
+func (s *RouterTestSuite) TestGetCustomQuote_GetCustomDirectQuotes_Mainnet_UOSMOUSDC() {
```

Comment:
Let's move this into `router_usecase_test.go`? I think we will separately need to move other tests later as well.

The current structure doesn't make much sense

Reply:
Moved `TestGetCustomQuote_GetCustomDirectQuotes_Mainnet_UOSMOUSDC` test case to `router_usecase_test.go`. Marking this thread as resolved.